File: Algorithmique appliquée/chapitre 1/exercice2.cpp
/*  programme table(exercice2)
*   but: affiche la table de 1 à 9 de la valeur entrée
*   date: 15/09/2016
*/
#include <stdio.h>
#include <iostream>
using namespace std;

int main(int argc, char *argv[]){

    unsigned int table, k;

    do{
        cout << "quelle table voulez-vous afficher entre 1 et 9? ";
        cin >> table;
    }while(table < 1 || table > 9);


    for(k=0;k<13;k++){
        cout << table << "*" << k << "=" << table*k << "\n";
    }
    return 0;
}

File: Algorithmique appliquée/chapitre 1/python/exercice2.py
# -*-coding:utf-8 -*
# Programme exercice 2
# affiche la table de 1 à 9 de la valeur entrée
# date : 15/09/2016


import os


k = 1

# saisie de la valeur avec contrôle de saisie
val = 0
while val < 1 or val > 9 :
    val = int(input("Entrez un nombre entier = "))

# affichage de la table de multiplication pour cette valeur
for k in range(1, 11) :
    print(val, " * ", k, " = ", (val*k), "\n")


os.system("pause")
